src/utils/formatDateForEditing.ts

```diff
@@ -0,0 +1,16 @@
+export const formatDateForEditing = (date: string) => {
+    const parts = date.split(' ');
+    const dateParts = parts[0].split('.');
+    const timeParts = parts[1].split(':');
+
+    const formattedYear = parseInt(dateParts[2]);
+    const formattedMonth = parseInt(dateParts[1]) - 1;
+    const formattedDay = parseInt(dateParts[0]);
+    const formattedHour = parseInt(timeParts[0]);
+    const formattedMinute = parseInt(timeParts[1]);
+    const formattedDate = new Date(formattedYear, formattedMonth, formattedDay, formattedHour, formattedMinute);
+
+    formattedDate.setHours(formattedDate.getHours() + 3);
```

Comment:
Magic number - 3 should be as const

And are you sure that +3 will work for another timezone ?